Uno.Gallery/Uno.Gallery.Shared/App.xaml.cs

```diff
@@ -226,28 +223,64 @@ public void SearchShellNavigateTo(Sample sample)
 			_shell.NavigationView.Content = page;
 		}
 
-		private Shell BuildShell()
+		private Shell BuildShell(LaunchActivatedEventArgs e)
 		{
 			_shell = new Shell();
 			AutomationProperties.SetAutomationId(_shell, "AppShell");
 			_shell.RegisterPropertyChangedCallback(Shell.CurrentSampleBackdoorProperty, OnCurrentSampleBackdoorChanged);
 			var nv = _shell.NavigationView;
 			AddNavigationItems(nv);
 
-			// landing navigation
-			ShellNavigateTo<OverviewPage>(
+			if (!IsThereSampleFilteredByArgs(nv, e))
```

Comment:
I would highly suggest redirecting to `/` instead of keeping an invalid fragment in the url.

Reply:
I understand that the user could have typed it wrong and would have the opportunity to correct it.
Redirecting to / would not allow this fix.

But, I implemented it as requested.
https://github.com/unoplatform/Uno.Gallery/commit/26d34686db9cd34a5e7227e8723af669f3c4493e